go/stellar/remote/remote.go

```diff
@@ -437,3 +437,32 @@ func SetAccountDefaultCurrency(ctx context.Context, g *libkb.GlobalContext, acco
 	_, err := g.API.Post(apiArg)
 	return err
 }
+
+type disclaimerResult struct {
+	libkb.AppStatusEmbed
+	AcceptedDisclaimer bool `json:"accepted_disclaimer"`
+}
+
+func GetUserSettings(ctx context.Context, g *libkb.GlobalContext) (stellar1.UserSettings, error) {
+	apiArg := libkb.APIArg{
+		Endpoint:    "stellar/disclaimer",
+		SessionType: libkb.APISessionTypeREQUIRED,
+		NetContext:  ctx,
+	}
+	var apiRes disclaimerResult
+	err := g.API.GetDecode(apiArg, &apiRes)
+	if err != nil {
+		return stellar1.UserSettings{AcceptedDisclaimer: false}, err
```

Comment:
just do
`return stellar1.UserSettings{}, err`
or better:
- change function signature to
`func GetUserSettings(ctx context.Context, g *libkb.GlobalContext) (res stellar1.UserSettings, err error) {`
- change this return to `return res, err`